go/Makefile

```diff
@@ -0,0 +1,49 @@
+solidity_dir=$(realpath ${SOLIDITY_DIR})
+solidity_files := $(foreach file,${SOLIDITY_FILES},$(file))
```

Comment:
Why don't we use the same mechanism as in `keep-core` and `keep-ecdsa`?
```
# Environment provides the solidity directory as a potentially-relative path,
# which we resolve. Then we resolve the Solidity files in a contracts/ directory
# at that path.
solidity_dir=$(realpath ${SOLIDITY_DIR})
solidity_files := $(wildcard ${solidity_dir}/contracts/*.sol)
```

Reply:
Because `tbtc` contracts are grouped into subdirectories.